marklogic-langchain4j/build.gradle

```diff
@@ -11,4 +11,8 @@ dependencies {
   // Supports using Semaphore to classify documents
   implementation 'com.smartlogic.csclient:Semaphore-CS-Client:5.6.1'
   implementation 'com.smartlogic.cloud:Semaphore-Cloud-Client:5.6.1'
+  compileOnly("com.fasterxml.jackson.dataformat:jackson-dataformat-xml:2.15.2") {
```

Comment:
This may need to be implementation - i.e. I don't think Spark typically contains this Jackson dependency.

Reply:
I tested via Flux with a "compileOnly" build and an "implementation" build. Both worked, but I'm changing it to "implementation" anyway, just in case.